tests/unit/routes/reminders-test.js

```diff
@@ -0,0 +1 @@
+
```

Comment:
For the most part route tests happen organically in acceptance tests we can delete this file